CONTRIBUTING.rst

```diff
@@ -165,12 +165,13 @@ For more detailed instructions and options, see the `VS Code documentation`_.
 Logging in with GitHub
 ----------------------
 
-To setup the GitHub OAuth integration, run the ``populate_social_apps``
-management command. The values to use in place of the ``XXX`` and ``YYY`` flags
-can be found in the GitHub App, or if you're an OddBird you can find these
-values in the shared Keybase team folder (``metashare/prod.db``)::
+To setup OAuth integration, run the ``populate_social_apps`` management command.
+The values to use in place of ``AAA``, ``BBB``, ``XXX`` and ``YYY`` can be found
+in the GitHub App and Salesforce App, respectively, or if you're an OddBird you
+can find these values in the shared Keybase team folder
+(``metashare/prod.db``)::
 
-    $ docker-compose run --rm web python manage.py populate_social_apps --gh-id XXX --gh-secret YYY
+    $ docker-compose run --rm web python manage.py populate_social_apps --gh-id AAA --gh-secret BBB --sf-id XXX --sf-secret YYY
```

Comment:
So the docs are written for someone setting up the project from scratch, in which case this command (which includes both `--sf-*` and `--gh-*) is correct. Since you (presumably) already had the GH connection setup, you only needed to run the `--sf-*` command.